data/helpers.d/psql

```diff
@@ -221,7 +221,7 @@ ynh_psql_remove_db() {
 	ynh_handle_getopts_args "$@"
 
 	local psql_root_password=$(sudo cat $PSQL_ROOT_PWD_FILE)
-	if ynh_psql_database_exists "$db_name"; then # Check if the database exists
+	if $(ynh_psql_database_exists "$db_name"); then # Check if the database exists
```

Comment:
Are you sure about this tho ? To me it means two different things ? o.O

Reply:
Indeed, with $() if will test the value of the string echoed by ynh_psql_database_exists. But there isn't any.
Without $(), if will test the exit code. So you probably don't want to add a sub shell here.

Reply:
This code is inspired by : https://github.com/YunoHost/yunohost/blob/stretch-unstable/data/helpers.d/mysql#L213

But ok, i'll change that.

Reply:
Well yes indeed ... but in fact that looks like a bug in the mysql helper ? (@maniackcrudelis do you agree ?)

```
function foo()
{
    return 1
}

echo $(foo)
```

doesn't print "1" in the console ...

Reply:
Yes agree on the principle. Here you would ask yourself, what do you want, the return code or any echo in the function ?